### ISSUE TITLE
slider on a hose

### ISSUE THREAD

**Issue by DaanDirk** [airbnb/lottie-web#650]
* **Environment:** windows
* **Browser and Browser Version:** firefox current
* **After Effects Version:** current

**What did you do? Please explain the steps you took before you encountered the problem.**
I'm trying to modify a rubberhose with a slider from joystick n sliders, so i can later dymanically change the hose.

**What did you expect to happen?**
It works in the bodymovin preview window in AE, but in firefox i doesn't.

**What actually happened? Please include as much _relevant_ detail as possible.**
I get the following console message in firefox:
TypeError: thisLayer(...) is not a function[Learn More] ...bodymovin4111.js%20line%2012261%20%3E%20eval:2:10
Which I guess leads to line 12261 in bodymovin: var fn = eval('[function(){' + val+';this.v = $bm_rt;}' + ']')[0];
from the initiateExpression function.
The slider expression works outside the hoses, but not combined with the hoses (in the hose layer).
It has this form:

> slider(value);
                //////BEGIN SLIDER CODE//////
                function slider (inputValue){
                var contrl = thisComp.layer("#TheoPiet");
                var contrlCurVal1 = contrl("ADBE Effect Parade")(1)("ADBE Slider Control-0001");
var k1=[30.5,5];
var k2=[116,19];
function unArray(inKey){                            if(inKey instanceof Array && inKey.length === 1){                                return inKey[0];                            }else{                                return inKey;                            }                        }k1= unArray(k1);
k2= unArray(k2);
if(contrlCurVal1 >=0){
 var resultAdd1 = linear(contrlCurVal1,0,100,k1,k2);}else{var resultAdd1 = k1+(-linear(-contrlCurVal1,0,100,k1,k2)+k1);}
var myResult = resultAdd1+(inputValue-k1);
return myResult;}

Is there an easy fix for this?
Much obliged!

**Comment by DaanDirk** [comment]
Turns out it was a simple reference to a slider on the hose itself.
> thisLayer("Effects")("Width")("Slider")

I used it to adjust the stroke thickness, but it gave the above error.

**Comment by bodymovin** [comment]
Is this working now?
If not, can you share the .aep?

**Comment by DaanDirk** [comment]
I got my project working by eliminating the sliders. :) I'll show it soon.

The problem is caused by two sliders on each hose which I used to set stroke width.
There's no need for them anymore because i found the right thickness.
But still, it was a bit surprising that these didn't work in the browser.

https://www.dropbox.com/s/c2uct6u2awjz637/2017%20Rechthoekig%20hose%20slider.aep?dl=0
http://www.daandirk.com/bm-test-data/
